Gemfile.lock

```diff
@@ -289,8 +289,5 @@ DEPENDENCIES
   twitter-bootstrap-rails
   web-console (~> 2.0)
 
-RUBY VERSION
-   ruby 2.2.2p95
```

Comment:
Is there a reason for this? Compare with [this](https://github.com/hpi-swt2/workshop-portal/pull/51#pullrequestreview-9704030)